generic/tests/unattended_install_reboot_driftfix.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from virttest import error_context
+
+from virttest.tests import unattended_install
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Unattended install test:
+    1) Starts a VM with an appropriated setup to start an unattended OS install.
+    2) Wait until the install reports to the install watcher its end.
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    params["cpu_model_flags"] = ""
+    unattended_install.run(test, params, env)
+    vm = env.get_vm(params.get("main_vm"))
+    error_context.context("Try to log into guest '%s'." % vm.name, logging.info)
+    session = vm.wait_for_login()
```

Comment:
Login windows vm need the rss/serial service support in guest, but these service need reboot vm after installation to take effect. That is why login vm failed as `connection refused`. 
As we discussed, let's shutdown the vm and boot it again for the following test steps. Thanks!